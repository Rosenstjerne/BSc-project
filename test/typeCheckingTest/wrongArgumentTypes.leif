function int f(int a, int b) {
    return a + b;
}

print( f(5, 8 == 8) );