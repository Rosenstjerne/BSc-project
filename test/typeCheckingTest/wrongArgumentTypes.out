
Error in phase Type Checking, line 5:
    Argument 1 in call to function f is of type 'bool', but must be of type 'int'
